README.md

```diff
@@ -38,9 +38,11 @@ _Want to add something? Open a PR!_ 🙂
 
 ### Stackoverflow [(link)](https://stackoverflow.com/questions/tagged/amazon-qldb)
 
- - [Computing commit digest](https://stackoverflow.com/questions/57937869/how-to-get-compute-commitdigest-when-committing-a-transaction-in-aws-qldb)
- - [Pagination in QLDB](https://stackoverflow.com/questions/59502337/pagination-in-qldb)
- - [QLDB unique field](https://stackoverflow.com/questions/59408019/qldb-unique-field)
+ - [Computing commit digest](https://stackoverflow.com/questions/57937869/how-to-get-compute-commitdigest-when-committing-a-transaction-in-aws-qldb) - explains the purpose and algorithm behind CommitDigest
```

Comment:
It will be interesting to see if a query could be run against stackoverflow to pull these back, although I presume you wont get meaning full headlines from it